.editorconfig

```diff
@@ -0,0 +1,140 @@
+root = true
```

Comment:
what is the purpose of this?

Reply:
It helps to apply some code standarts automatically, for developers who use one of popular IDEs in their work.